DBNostalgia/UnitOfWork.cs

```diff
@@ -85,6 +85,30 @@ void iterateAndFetch(IDbCommand command)
             return output;
         }
 
+        public T GetOne<T>(string procedure, Func<IDataReader, T> fetchClosure, ParametersBuilder parametersBuilder = null)
```

Comment:
Once we get rid of the transaction vs non-transaction duplication this non-dry approach can be removed.